### DISCUSSION TITLE
How to assess contribution to final impact based on different levels/tiers of processes

### DISCUSSION THREAD

**Discussion by ketanvaidya** [LCA-ActivityBrowser/activity-browser#1406] [Q&A] [answered]
Hello! Kind of new here trying to get a grasp of AB so apologies if it is a very fundamental question. 
Starting of appreciate and respect the development until now it has been very good. I have modelled a process and analysed results. Now I need to figure out a way to summarize the results as per the contributing processes. I can see this in Sankey diagram but I am unable to export it/summarize it from the process contributions section. 
Lets say I am looking at an end product now I want to understand what are the key contributors if I aggregate the input to 1 tier downstream to see which are the actors the end producer can influence. 
Any help is greatly appreciated!
Thanks, Ketan

**Comment by marc-vdm** [comment]
Hi Ketan,

This is exactly the right place to ask! What you're asking for (the first tier contributors) is not yet available in AB, however, we will have this in the next minor release (2.11), you can subscribe to #810 to receive a notification when this feature becomes available and you can follow progress on us building it in #1046 if you're interested.

Of course, that is not useful for you right now. For now you unfortunately will have to manually record all numbers from the Sankey and make your own graphs. Do also check out our [wiki page on contribution analysis](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/LCA-Results#contribution-analysis) to learn more about what you can and cannot do in AB.

**Comment by ketanvaidya** [comment]
Hi Marc, Thanks for the prompt response. Really interesting to see the progress and development of this overtime. Due to being quite new to this maybe it is a bit early for me to start contributing but I will try my best to support the progress.

**Reply by marc-vdm** [reply]
Happy to help! Don't worry about your experience with AB or with coding, we always encourage contributions from the community!

**Comment by marc-vdm** [comment, marked as answer]
@ketanvaidya 
This is now possible with [First Tier contributions](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/LCA-Results#first-tier-ft-contributions), which is available since Activity Browser [`2.11.0`](https://github.com/LCA-ActivityBrowser/activity-browser/releases/tag/2.11.0).
